src/Microsoft.Health.Fhir.Api/Features/ActionResults/OperationResult.cs

```diff
@@ -0,0 +1,85 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Net;
+using EnsureThat;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Health.Fhir.Core.Features.Export;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Api.Features.ActionResults
+{
+    /// <summary>
+    /// Used to handle the result of an operations. Currently supports only Export.
+    /// </summary>
+    public class OperationResult : ActionResult
+    {
+        public OperationResult(HttpStatusCode statusCode)
+        {
+            StatusCode = statusCode;
+        }
+
+        public OperationResult(ExportJobResult jobResult, HttpStatusCode statusCode)
+        {
+            EnsureArg.IsNotNull(jobResult, nameof(jobResult));
+
+            JobResult = jobResult;
+            StatusCode = statusCode;
+        }
+
+        public HttpStatusCode StatusCode { get; set; }
+
+        public ExportJobResult JobResult { get; }
```

Comment:
The class name seems to indicate this is reusable for operations but the export job result is hard-coded here.

Reply:
Changed it to `ExportResult`